solr/solrj/src/java/org/apache/solr/common/params/QueryElevationParams.java

```diff
@@ -59,4 +59,11 @@ public interface QueryElevationParams {
    * set this to true. False by default.
    */
   String ELEVATE_ONLY_DOCS_MATCHING_QUERY = "elevateOnlyDocsMatchingQuery";
+
+  /**
+   * By default, the component respects the fq parameter. If you want to elevate documents that do
+   * not match the provided filters, tag the filters in question via the local parameter syntax
+   * fq={!tag=t1}field1:value1 and then specify the tags for exclusion via elevate.excludeTag=t1
```

Comment:
```suggestion
   * fq={!tag=t1}field1:value1 and then specify the tags for exclusion via elevate.excludeTags=t1
```

Reply:
It's sad this component didn't use a common prefix naming standard like the other components; ah well.

Reply:
Looking at the other parameter names, I see that some include a form of "elevate" (`enableElevation`, `forceElevation`, `useConfiguredElevatedOrder`, `elevateIds`) and others don't (`exclusive`, `markExcludes`, `excludeIds`). None of them include dot. Is it better to use a prefix naming standard for all new params, starting now, or try to be consistent with what's already there? I'm not seeing much of a pattern in the existing param names to try to match, so I'll plan to leave this as `elevate.excludeTags` unless others have feedback?

Reply:
I argue the current names are sloppy tech-debt that should change.  Doing that is out of scope of this PR for sure.  I recognize, this parameter looks different until this is rectified.  A comment there could acknowledge this.

Reply:
Agree... added a comment.